client/keys/add.go

```diff
@@ -100,8 +105,30 @@ func runAddCmd(cmd *cobra.Command, args []string) error {
 			}
 		}
 
+		multisigKeys := viper.GetStringSlice(flagMultisig)
+		if len(multisigKeys) != 0 {
+			var pks []crypto.PubKey
+
+			multisigThreshold := viper.GetInt(flagMultiSigThreshold)
+			if err := validateMultisigThreshold(multisigThreshold, len(multisigKeys)); err != nil {
+				return err
+			}
+
+			for _, keyname := range multisigKeys {
+				k, err := kb.Get(keyname)
+				if err != nil {
+					return err
+				}
+				pks = append(pks, k.GetPubKey())
+			}
+
+			pk := multisig.NewPubKeyMultisigThreshold(multisigThreshold, pks)
+			_, err := kb.CreateOffline(name, pk)
```

Comment:
Can we print out something, like "Saved key to disk"?

Reply:
Sure we can: https://github.com/cosmos/cosmos-sdk/pull/3264/commits/050bb1b93e159cd140928efe1b95e16f5d938d71